aten/src/ATen/native/cuda/block_reduce.cuh

```diff
@@ -34,6 +36,55 @@ __inline__ __device__ T BlockReduceSum(T val, T* shared) {
   return val;
 }
 
+template <typename T>
+__inline__ __device__ thrust::tuple<int64_t, T, T> WarpReduceMoments(
```

Comment:
is there any reason you are not using WelfordOps.combine from SharedReduceOps.h here?

Reply:
Changed to WelfordOps